docs/_index_included.yaml

```diff
@@ -0,0 +1,157 @@
+included:
+- heading: ReCirq Guide
+  description: How to use and contribute to ReCirq.
+  items:
+  - heading: Research with ReCirq
+    description: How to use this code.
+    path: /cirq/experiments/research
+    icon:
+      name: menu_book
+  - heading: ReCirq experiment structure
+    description: How ReCirq experiments are structured.
+    path: /cirq/experiments/guide/data_collection_idioms
+    icon:
+      name: menu_book
+  - heading: Data collection
+    description: How ReCirq experiments collect data.
+    path: /cirq/experiments/guide/data_collection
+    icon:
+      name: menu_book
+  - heading: Data analysis
+    description: How ReCirq experiments analyze data.
+    path: /cirq/experiments/guide/data_analysis
+    icon:
+      name: menu_book
+  - heading: Best practices
+    description: Additional advice on writing ReCirq experiments.
+    path: /cirq/experiments/guide/best_practices
+    icon:
+      name: menu_book
+
+- heading: QAOA
+  description: Tutorials and examples with the Quantum Approximate Optimization Algorithm.
+  items:
+  - heading: Overview
+    path: /cirq/experiments/qaoa
+    icon:
+      name: menu_book
+  - heading: Example problems
+    description: Introductory QAOA problems.
+    path: /cirq/experiments/qaoa/example_problems
+    icon:
+      name: menu_book
+  - heading: Tasks
+    description: QAOA experiment Task object.
+    path: /cirq/experiments/qaoa/tasks
+    icon:
+      name: menu_book
+  - heading: Precomputed analysis
+    description: QAOA analysis with precomputed optimal angles.
+    path: /cirq/experiments/qaoa/precomputed_analysis
+    icon:
+      name: menu_book
+  - heading: Landscape analysis
+    description: QAOA analysis with a hardware landscape.
+    path: /cirq/experiments/qaoa/landscape_analysis
+    icon:
+      name: menu_book
+  - heading: Optimization analysis
+    description: QAOA analysis with an optimization path.
+    path: /cirq/experiments/qaoa/optimization_analysis
+    icon:
+      name: menu_book
+  - heading: Hardware grid circuits
+    description: QAOA on grid-shaped hardware.
+    path: /cirq/experiments/qaoa/hardware_grid_circuits
+    icon:
+      name: menu_book
+  - heading: Routing with tket
+    description: Use t|ket> to map QAOA to a generic device.
+    path: /cirq/experiments/qaoa/routing_with_tket
+    icon:
+      name: menu_book
+
+- heading: Quantum Chess
+  description: Tutorials and examples with the Quantum Approximate Optimization Algorithm
+  items:
+  - heading: Overview
+    path: /cirq/experiments/quantum_chess
+    icon:
+      name: menu_book
+  - heading: Concepts
+    description: What is Quantum Chess?
```

Comment:
I don't think this is an accurate description of this page.  I would say something like `Concepts needed to adapt quantum chess rules onto a physical device`